force-app/main/kontaktskjema/classes/KontaktSkjemaHandlerTest.cls

```diff
@@ -756,7 +756,7 @@ private without sharing class KontaktSkjemaHandlerTest {
 
```

Comment:
```suggestion

@isTest
private static void accountIsEmptyWhenOrgNoMissing() {
}

```

Ønsker at du lager denne testen og sørger for at den blir rød, før du fikser feilen med at "Kontakter uten konto" blir satt direkte på muligheten.